.gitignore

```diff
@@ -13,7 +13,8 @@ coverage
 .DS_Store
 dist
 **-changes-cache.json
-
+.idea/**/*
+.idea/*
```

Comment:
We are currently intentionally syncing all the idea files other than the ones are ignored here to reuse: Nodejs configs, Run Configuraitons and lining rules across team members. Any specific reasons to ignore it?

Reply:
My webstorm was adding new files that were not a part of the project and I was trying to eliminate side effects, I’ll check what they are and share, so you can let me know what is preferred
I usually do that because the project settings may differ caused by personal preference and didn’t want to impose mine